devices/HapticGlove/include/SenseGloveHelper.hpp

```diff
@@ -0,0 +1,250 @@
+/**
+ * @file SenseGloveHelper.hpp
+ * @authors Kourosh Darvish <kourosh.darvish@iit.it>
+ * @copyright 2021 iCub Facility - Istituto Italiano di Tecnologia
+ *            Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+ * @date 2021
+ */
+
+#ifndef SENSE_GLOVE_HELPER_HPP
+#define SENSE_GLOVE_HELPER_HPP
+
+#include <Eigen/Dense>
+
+// std
+#include <memory>
+#include <iostream>
+
+// YARP
+#include <yarp/sig/Vector.h>
+#include <yarp/os/LogStream.h>
+
+// Sense Glove
+#include <DeviceList.h>
+#include <SenseGlove.h>
+
+/**
+ * GloveControlHelper is an helper class for controlling the glove.
+ */
+namespace senseGlove
+{
+    class SenseGloveHelper;
+    const std::string LogPrefix = "senseGlove::SenseGloveHelper::";
+}
+
+class senseGlove::SenseGloveHelper
+{
+    int m_forceFbDof; /**< Number of the actuated motors Dofs to produce force feedback to the human*/
```

Comment:
What I mean is the following:

```suggestion
    /// Number of the actuated motors Dofs to produce force feedback to the human
    int m_forceFbDof;
```

Otherwise, clang-format will mess everything up as soon as you will process all PR sources.